libs/facilities/frontend/view/src/lib/components/browse/browse.page.html

```diff
@@ -1,87 +1,116 @@
-<div class="absolute top-20 right-4 !z-[50] flex flex-row-reverse gap-x-4">
-    <ix-button (click)="toggleView()">
-        Switch to {{ showCardList() ? 'Event List' : 'Card List' }}
-    </ix-button>
-    <ix-button (click)="toggleFilter()">
-        {{ filterIssues() ? 'Remove Filter for Issues' : 'Filter Facilities with Issues' }}
-    </ix-button>
-</div>
+@if (filterIssues()) {
+    <ix-content-header
+        class="pl-4 pt-4"
+        hasBackButton="true"
+        headerTitle="Facilities Overview"
+        headerSubtitle="View all facilities that are not running properly"
+        (backButtonClick)="location.back()"
+    ></ix-content-header>
 
-<ix-content-header
-    class="pl-4 pt-4"
-    hasBackButton="true"
-    headerTitle="Facilities Overview"
-    headerSubtitle="View all facilities and their status"
-    (backButtonClick)="location.back()"
-></ix-content-header>
+    <ix-empty-state
+        class="mt-8"
+        header="No facilities with issues"
+        subHeader="All facilities are running properly. Switch to the full list to view all facilities."
+        icon="check"
+        action="Switch to full list"
+        (actionClick)="toggleFilter()"
+    ></ix-empty-state>
+} @else {
+    <ix-content-header
+        class="pl-4 pt-4"
+        hasBackButton="true"
+        headerTitle="Facilities Overview"
+        headerSubtitle="View all facilities and their status"
+        (backButtonClick)="location.back()"
+    ></ix-content-header>
+}
+@if (!allFacilities()) {
+    <ix-empty-state
+        class="mt-8"
+        header="No facilities available"
+        icon="document-fail"
+    ></ix-empty-state>
+} @else {
+    <div class="absolute top-20 right-4 !z-[50] flex flex-row-reverse gap-x-4">
+        <ix-button (click)="toggleView()">
+            Switch to {{ showCardList() ? 'Event List' : 'Card List' }}
+        </ix-button>
+        <ix-button (click)="toggleFilter()">
+            {{ filterIssues() ? 'Remove Filter for Issues' : 'Filter Facilities with Issues' }}
+        </ix-button>
+    </div>
```

Comment:
I think we can have these buttons outside the `@if` too because it is weird now that you can toggle the filter off right now when there are no facilities but you can never toggle it back on